generic_tracers/generic_COBALT.F90

```diff
@@ -399,67 +399,98 @@ subroutine user_add_params(param_file)
-    call get_param(param_file, "generic_COBALT", "htotal_scale_lo", cobalt%htotal_scale_lo, "htotal_scale_lo", units=" ", default=0.01)
-    call get_param(param_file, "generic_COBALT", "htotal_scale_hi", cobalt%htotal_scale_hi, "htotal_scale_hi", units=" ", default=100.0)
-
-    !  Rho_0 is used in the Boussinesq
-    !  approximation to calculations of pressure and
-    !  pressure gradients, in units of kg m-3.
-    call get_param(param_file, "generic_COBALT", "RHO_0", cobalt%Rho_0, "RHO_0", units=" ", default=1035.0)
-    call get_param(param_file, "generic_COBALT", "NKML" , cobalt%nkml,  "NKML" , units=" ", default=1)
-    !-----------------------------------------------------------------------
-    !       coefficients for O2 saturation
-    !----------------------------------------------------------------------
-    call get_param(param_file, "generic_COBALT", "a_0", cobalt%a_0, "a_0", units=" ", default=2.00907)
-    call get_param(param_file, "generic_COBALT", "a_1", cobalt%a_1, "a_1", units=" ", default=3.22014)
-    call get_param(param_file, "generic_COBALT", "a_2", cobalt%a_2, "a_2", units=" ", default=4.05010)
-    call get_param(param_file, "generic_COBALT", "a_3", cobalt%a_3, "a_3", units=" ", default= 4.94457)
-    call get_param(param_file, "generic_COBALT", "a_4", cobalt%a_4, "a_4", units=" ", default= -2.56847e-01)
-    call get_param(param_file, "generic_COBALT", "a_5", cobalt%a_5, "a_5", units=" ", default= 3.88767)
-    call get_param(param_file, "generic_COBALT", "b_0", cobalt%b_0, "b_0", units=" ", default= -6.24523e-03)
-    call get_param(param_file, "generic_COBALT", "b_1", cobalt%b_1, "b_1", units=" ", default= -7.37614e-03)
-    call get_param(param_file, "generic_COBALT", "b_2", cobalt%b_2, "b_2", units=" ", default= -1.03410e-02 )
-    call get_param(param_file, "generic_COBALT", "b_3", cobalt%b_3, "b_3", units=" ", default= -8.17083e-03)
-    call get_param(param_file, "generic_COBALT", "c_0", cobalt%c_0, "c_0", units=" ", default= -4.88682e-07)
-    !-----------------------------------------------------------------------
-    !     Schmidt number coefficients
-    !-----------------------------------------------------------------------
+    call get_param(param_file, "generic_COBALT", "htotal_scale_lo", cobalt%htotal_scale_lo, &
+                   "scaling factor for initializing carbon chemistry solver", units=" ", default=0.01)
+    call get_param(param_file, "generic_COBALT", "htotal_scale_hi", cobalt%htotal_scale_hi, &
+                   "scaling factor for initializing carbon chemistry solver", units=" ", default=100.0)
+
+    call get_param(param_file, "generic_COBALT", "RHO_0", cobalt%Rho_0, "reference density", &
+                   units="kg m-3", default=1035.0)
+    !------------------------------------------------------------------------------------------------------------------
+    ! Coefficients for O2 saturation based on Garcia and Gordon, 1992.  Oxygen solubility in seawater: Better fitting  
```

Comment:
It looks like these are the coefficients for Garcia and Gordon's fit to only the Benson and Krause data (first columns of Table 1). Is it worth updating to the combined fit?

Reply:
I will take this up with John Dunne and Jasmin John to discuss whether there was a rationale for this.  For now, I will add a bit to the comment to indicate that it uses the fit to the Benson and Krause data.  I'll add a new commit with this addition shortly.